src/js/pages/Search/index.js

```diff
@@ -39,21 +39,23 @@ export default class Search extends React.PureComponent {
         {({ loading, data }) => {
           if (loading || data === undefined) return <Stub />;
 
-          const { pageinfo, shabads } = data;
+          const { resultsInfo, verses } = data;
 
           return (
             <Layout
               pages={Array.from(
                 Array(
-                  parseInt(pageinfo.totalresults / pageinfo.resultsperpage)
+                  parseInt(
+                    resultsInfo.totalResults / resultsInfo.pages.resultsPerPage
+                  )
                 ),
                 (_, i) => i + 1
               )}
-              totalResults={pageinfo.totalresults || 0}
-              resultsCount={pageinfo.pageresults || 0}
+              totalResults={resultsInfo.totalResults || 0}
+              resultsCount={resultsInfo.pageResults || 0}
               offset={offset}
-              nextPageOffset={pageinfo.nextpageoffset}
-              shabads={shabads}
+              //nextPageOffset={resultsInfo.pages.page}
```

Comment:
This isn't needed anymore?

Reply:
Yeah, forgot to ask, this wasn't used anywhere. Looked like some old code, we forgot to remove.
And, also it's not in the response of v2 API.